src/PostgREST/Config.hs

```diff
@@ -395,6 +394,16 @@ parser optPath env dbSettings roleSettings roleIsolationLvl =
         Nothing   -> pure Nothing
         Just orig -> pure $ Just (T.strip <$> T.splitOn "," orig)
 
+    parseDbSchemas :: C.Key -> C.Key -> C.Parser C.Config (NonEmpty Text)
+    parseDbSchemas k al =
+      optWithAlias (optString k) (optString al) >>= \case
+        Nothing      -> pure $ fromList ["public"]
+        Just ""      -> pure $ fromList ["public"]
```

Comment:
Isn't this `db-schemas = ""`? Shouldn't it be a failure in this case?

Reply:
We were defaulting to `public` on just `""`. Failing on this required fixing too many io-test cases that started failing after this change. I'll recheck and confirm.